src/pages/mesh/release/index.md

```diff
@@ -18,6 +18,20 @@ import UpdateNotice from '/src/_includes/update-notice.md'
 
 The following sections list updates to API Mesh for Adobe Developer App Builder. Refer to the [Upgrade version](upgrade.md) for more information on upgrading.
 
+## April 22, 2025
+
+This release contains the following changes to API Mesh:
+
+### Enhancements
+
+- This release introduces [native caching](../advanced/caching/index.md#enable-native-caching) for API Mesh, including how to [verify caching behavior using response headers](../advanced/caching/index.md#verifying-the-caching-behavior-using-response-headers). It also provides guidance on [source-driven caching](../advanced/caching/index.md#source-driven-caching).
+- Logging now includes HTTP details. To see these changes, perform a [mesh update](../basic/create-mesh.md#update-an-existing-mesh).
+
+### Bug fixes
+
+- [Deleting a mesh](../basic/create-mesh.md#delete-a-mesh) will now also delete the mesh's [logging configuration](../advanced/logging.md).
+- Resolved an issue that could cause above average response times.
```

Comment:
I don't think we need to mention this. This is on a health check endpoint that only we use so I think we can omit this one